sdk/core/azure-core-amqp/src/models/amqp_header.cpp

```diff
@@ -14,6 +14,8 @@
 #include <azure_uamqp_c/amqp_definitions_header.h>
 #elif ENABLE_RUST_AMQP
 #include "azure/core/amqp/internal/common/runtime_context.hpp"
+
+#include <numeric>
```

Comment:
I don't see any of the numeric functions used, at least not in the lines you are changing.